model/client.go

```diff
@@ -605,8 +605,15 @@ func (c *Client) JoinGroup(groupID, installationID string) error {
 }
 
 // LeaveGroup removes an installation from its group, if any.
-func (c *Client) LeaveGroup(installationID string) error {
-	resp, err := c.doDelete(c.buildURL("/api/installation/%s/group", installationID))
+func (c *Client) LeaveGroup(installationID string, request *LeaveGroupRequest) error {
+	u, err := url.Parse(c.buildURL("/api/installation/%s/group", installationID))
+	if err != nil {
+		return err
```

Comment:
Would there be any benefit to wrapping this error?

Reply:
Probably, noticed we don't really wrap errors much in the API package we also typically log in lowercase letters for w/e reason. lol

Maybe the package should just have a follow-up clean and better-logging PR. Good feedback though.